src/main/kotlin/io/snyk/plugin/ui/jcef/ThemeBasedStylingGenerator.kt

```diff
@@ -57,20 +58,21 @@ class ThemeBasedStylingGenerator {
                 httpStatusCode: Int,
             ) {
                 if (frame.isMain) {
-                    val baseColor = UIUtil.getTextFieldBackground()
-                    val (addedColor, removedColor) = getCodeDiffColors(baseColor, isHighContrast)
+                    val baseColor = UIUtil.getTextFieldBackground() //TODO Replace with JBUI.CurrentTheme colors
+                    val (addedColor, removedColor) = getCodeDiffColors(baseColor, isHighContrast) //TODO Replace with JBUI.CurrentTheme colors
```

Comment:
Same here? Is the TODO still right?

Reply:
@neil-snyk will provide the new colors from the JBUI color schemas.